set.mm

```diff
@@ -496903,6 +497001,32 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
+  ${
+    $d B e p x y $.  $d S e p x y $.  $d T e p x y $.  $d e p ph x y $.
+    urpropd.b $e |- B = ( Base ` S ) $.
+    urpropd.s $e |- ( ph -> S e. V ) $.
+    urpropd.t $e |- ( ph -> T e. W ) $.
+    urpropd.1 $e |- ( ph -> B = ( Base ` T ) ) $.
+    urpropd.2 $e |- ( ( ( ph /\ x e. B ) /\ y e. B )
+                                -> ( x ( .r ` S ) y ) = ( x ( .r ` T ) y ) ) $.
+    $( Sufficient condition for ring identities to be equal.  (Contributed by
```

Comment:
We wanted to call the multiplicative identities in a ring "ring unity".

Reply:
Done in https://github.com/metamath/set.mm/pull/4702/commits/cba959dc98515e7a4ee85cdfab98c8e7eb598b54 (as in other places)